packages/adp-tooling/src/writer/changes/writers/annotations-writer.ts

```diff
@@ -37,7 +37,7 @@ export class AnnotationsWriter implements IWriter<AnnotationsData> {
             annotationsInsertPosition: 'END',
             dataSource: {
                 [annotationNameSpace]: {
-                    uri: `../annotations/${fileName}`,
+                    uri: `../changes/annotations/${fileName}`,
```

Comment:
Actually, just understood that this change was not tested and we don't know how it will behave when deployed. Therefore, we will have a sync with a relevant team today to discuss this. If we are in a rush to merge this it is better this change to not be included in this PR.

Reply:
This change is required, otherwise the quick action or command is pointless, because the annotation file then would be ignored since it can't be located.